pibooth/plugins/lights_plugin.py

```diff
@@ -2,83 +2,100 @@
 
 import time
 import pibooth
+from pibooth.utils import LOGGER
+from gpiozero import Device, LEDBoard
+try:
+    from RPi import GPIO
+    LOGGER.info("Start on Raspberry pi")
+except ImportError:
+    # GPIO library only existes on Raspberry Pi
+    # Set the default pin factory to a mock factory
+    from gpiozero.pins.mock import MockFactory
+    Device.pin_factory = MockFactory()
+    LOGGER.info("Programme not start on Raspberry pi use MockFactory() mode")
 
 
 class LightsPlugin(object):
 
     """Plugin to manage the lights via GPIO.
     """
+    def __init__(self):
+        self.blink_time = 0.3
+        self.load = False
+
+    def start(self, cfg):
+
+        if not self.load:
+            self.leds = LEDBoard(
```

Comment:
LEDs shall remains available from other user plugins. The LEDBoard shall be created in the `booth.py` file.
This permits that all plugins can access to it using `app.leds`